### DISCUSSION TITLE
Sonoff POW Elite 20A (POWR320D) - Read voltage before relay ON

### DISCUSSION THREAD

**Discussion by coldfire10** [arendst/Tasmota#18437] [Q&A] [answered]
### PROBLEM DESCRIPTION
I have a problem with read correct voltage before relay is ON. When relay is ON voltage are correct. I'd just calibrate the power sensor and set SetOption21 to TRUE. But there is no effect.. I tested many version firmare but there is not working too.. Maybe this module don't support this function? I dont't know. Maybe whos have thesame problem?

I need to read actual voltage before relay is ON becouse I need set a rule something like: "when voltage is grether than 250V then turn on something..."

Thank's all for any help!

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [X ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [ X] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [X ] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [X ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [X ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [X ] Device used: Sonoff POW Elite 20A (POWR320D) 
- [ X] Tasmota binary firmware version number used: 12.5.0
  - [ ] Pre-compiled
  - [ ] Self-compiled
- [X ] Flashing tools used: Tasmota Installer powered by ESP Web Tools
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS

![on](https://user-images.githubusercontent.com/130984857/232473369-a5597db5-0bdf-4a4e-a0a3-88127354ace5.jpg)
![off](https://user-images.githubusercontent.com/130984857/232473394-c8733fff-4851-42b4-99ad-d6f2a58143e5.jpg)

### ADDITIONAL CONTEXT
I have used template from: https://templates.blakadder.com/sonoff_POWR320D.html
{"NAME":"Sonoff POWR320D","GPIO":[32,0,9313,0,9312,576,0,0,0,0,9280,0,3104,0,320,0,0,0,0,0,0,9184,9248,9216,0,0,0,0,0,0,0,0,0,0,0,0],"FLAG":0,"BASE":1}

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by sfromis** [comment, marked as answer]
To me, the board looks like the sensor part is on the output side of the relay, hence no monitoring without anything there.

BTW. Creating an "issue" is meant for things to be fixed, not about asking question.
